ambari-server/src/main/resources/common-services/AMBARI_METRICS/0.1.0/package/scripts/params.py

```diff
@@ -20,9 +20,11 @@
 
 import ConfigParser
 import os
+import re
 
 from ambari_commons import OSCheck
 from ambari_commons.ambari_metrics_helper import select_metric_collector_hosts_from_hostnames
+from resource_management import Script
```

Comment:
I'm not sure this is needed, since `Script` is imported via `params_linux`

Reply:
I've removed it